schema/src/main/java/org/hl7/v3/RCMRMT030101UK04Authorise.java

```diff
@@ -327,6 +327,17 @@ public boolean hasQuantity() {
         return this.predecessor;
     }
 
+    public RCMRMT030101UK04Predecessor getPredecessorFirstRep() {
```

Comment:
Does this Authorise class and the Predecessor class need added to the manually edited classes in the schema README?

Reply:
Authorise is already in the readme, added Predecessor.